include/RAJA/index/IndexSet.hpp

```diff
@@ -150,18 +162,20 @@ class TypedIndexSet<T0, TREST...> : public TypedIndexSet<TREST...>
   /// This is used to implement the == and != operators
   ///
   template <typename P0, typename... PREST>
-  RAJA_INLINE bool compareSegmentById(
-      size_t segid,
-      const TypedIndexSet<P0, PREST...> &other) const
+  RAJA_INLINE bool
```

Comment:
Why does it put a line break here between the return type and method name, but not below at line 189?